14-11-2018/monika/rot13/test_ror13.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'ror13'
+require 'test/unit'
+
+class TestRor13 < Test::Unit::TestCase
+
+  def test_change1
```

Comment:
I am a fan of descriptive methods/variables names. Try to be more descriptive when naming methods. It's a good habit.